scripts/zones/Sea_Serpent_Grotto/MobIDs.lua

```diff
@@ -5,13 +5,10 @@
 -----------------------------------
 
 -- Charybdis
-Charybdis = 17498522;
 Charybdis_PH =
 {
-    [1] = 17498520,
-    [2] = 17498518,
-    [17498520] = '1', -- -138.181, 48.389, -338.001
-    [17498518] = '1', -- -212.407, 38.538, -342.544
+    [17498518] = 17498522;  -- -138.181, 48.389, -338.001
```

Comment:
You need a comma after rows that aren't the end of the table

Reply:
I've been trying to figure out where the comma goes on this one for a while now. I apologize but it's just eluding me. I think you're referring to where the old code has commas but the new code has semicolons? Forgive me but, I thought it could be either or. I just followed Lotus' example:
https://github.com/DarkstarProject/darkstar/blob/9efe30a3f86f4b0787e3fb2146f0e1d8efd820c1/scripts/zones/South_Gustaberg/MobIDs.lua#L8

Reply:
[17498518] = 17498522,  -- -138.181, 48.389, -338.001

is what you want

the final line won't need it